app/controllers/portal/tax_returns_controller.rb

```diff
@@ -57,7 +56,7 @@ def permitted_params(form_class)
     def redirect_unless_signature_required(signature_type)
```

Comment:
In cases where the client is authenticated but they aren't able to take the action, we want to redirect to their portal dashboard instead of the homepage -- this was a non-blocking request by @alaroia-cfa on prev PR